config/m4/verbs.m4

```diff
@@ -186,7 +186,7 @@ if test "x$vma_cv_verbs" == x2; then
 
     AC_MSG_CHECKING([for multi packet RQ support])
     AS_IF([test "x$have_mp_rq" == xyes -a "x$enable_exp_cq" == xyes -a "x$vma_cv_directverbs" == x2],
-	    [AC_DEFINE([HAVE_MP_RQ], 1, [MP_RQ QP supported])] [AC_MSG_RESULT([yes])],
+	    [AC_DEFINE([HAVE_MP_RQ], 1, [MP_RQ QP supported])] [AC_MSG_RESULT([yes (warning: this feature is deprecated and will be removed in a future release)])],
```

Comment:
Is it **already** deprecated or will it be deprecated in future releases?

Reply:
It is deprecated but exist for 8.9.x and will be removed after